homeassistant/components/emoncms/strings.json

```diff
@@ -21,7 +21,9 @@
       }
     },
     "abort": {
-      "already_configured": "This server is already configured"
+      "already_configured": "This server is already configured",
+      "unique_id_mismatch": "Unique ID mismatch",
```

Comment:
Let's explain more what a unique id mismatch means, because the user does not work with a unique id

Reply:
done